cost/budget_report_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.2
+
+- Added a input user filter to group filter item request report object, dimension=value
```

Comment:
This should be rephrased to be a bit more clear. I had to read it a couple times to parse it.

Reply:
Changed to this.

- Added a parameter to define target groups to track by providing specific dimension values 'Cloud Vendor=AWS', 'Service=AmazonEC2'